### ISSUE TITLE
Publish an Open Leadership Definition

### ISSUE THREAD

**Issue by semioticrobotic** [open-organization/editorial#94]
This meta issue tracks all items related to our work on an Open Leadership Definition.

Issues relate to various stages of the editorial process and include:

- [x] Draft the document (#67)
- [x] Circulate the document for community review and feedback (#67 again)
- [x] Revise the document in response to community feedback (#93)
- [x] Revise the document to ensure accessibility to the widest possible audience (#93 again)
- [x] Compose a mission/vision statement for the project so others understand its goals (#71)
- [x] Prepare repositories to house the document (#119)
- [x] Upload a release candidate to the repositories for public review (#119 again)
- [x] Update reviewers and conduct final review (go/no-go) of document in repository ([see below](https://github.com/open-organization/editorial/issues/94#issuecomment-897751278))
- [x] Write release announcements (tracking in #121)
- [x] Finalize preparation of promotional materials (also in #121)
- [x] Prepare website for launch (underway [here](https://github.com/open-organization/open-organization.github.io/pull/22))
- [x] Launch the Open Leadership Definition (use materials and checklist in #121)

We'll continue to update this issue as deadlines draw nearer.

**Comment by semioticrobotic** [comment]
Updating this meta-checklist to note that [the public draft](https://drive.google.com/drive/folders/1e1N_0p5lJEwAo_s6hQ3OK0KaJIfc7fgF) is now released and has been reviewed.

**Comment by LauraHilliger** [comment]
started a blog post, edit at will > https://docs.google.com/document/d/1zgWLY2SU-mI6iOWeO4JUASY9FxEFeCLkyGzJ96kHwxA/edit#

**Comment by semioticrobotic** [comment]
Okay! Thanks, @LauraHilliger. Will do. Might be a day or two.

**Comment by HeidiHVL** [comment]
To finish the OLD and all the launch tasks: 
Tues 8/10 - (Workgroup meeting) Review the release/vision doc, Create repository and showcase in Workgroup for feedback, plan/assign release communications 
Weds 8/11 - Target to send email thank you to community, and that the OLD is in the repository for a last inspection
Thurs 8/12 - Assign any other release comms items to Ambassadors 
Weds 8/18 - repository inspection ends/any feedback is integrated
Mon 8/23 - launch!!

**Comment by semioticrobotic** [comment]
Thanks, @HeidiHVL, our expert project manager. This all looks good and makes sense to me. I will get to work updating our various issues, appending dates, and organizing the materials accordingly.

**Comment by LauraHilliger** [comment]
@semioticrobotic started formatting in the repo, made some commits. Hope I did it the way you would have!! https://github.com/open-organization/open-leadership-definition/blob/main/open-leadership-definition.md

**Comment by semioticrobotic** [comment]
You rock. Love it. Thank you.

**Comment by HeidiHVL** [comment]
Open a ticket for promotion of items after github release so that the community can be involved in the promotional announcements.

**Comment by semioticrobotic** [comment]
> Open a ticket for promotion of items after github release so that the community can be involved in the promotional announcements.

Done! See #121. Thanks for logging this, @HeidiHVL.

**Comment by HeidiHVL** [comment]
All, 

**TL;DR: Thank you for your feedback on the draft of the Open Leadership Definition, which was integral to its development. The definition is now available in GitHub for final review until Thursday August 19 at 5pm EST.** 

**Thank you**
Thank you for adding feedback and thoughts to the Open Leadership Definition! Your time and attention has helped create the most thoughtful and thorough definition of what it means to be an open leader. Throughout our review, we identified several themes of comments, some of which generated [lively discussions](https://opensource.com/open-organization/21/7/manage-feedback-open-project) within the workgroup. You can review a record of the comments, along with actions and notes, in the [Open Leadership Feedback report spreadsheet](https://docs.google.com/spreadsheets/d/1ETyMtoNK9MpkTOm2wUvqBBtcnf1S6wGWOUPvOYFyrx8/edit?usp=sharing).

**Next step: Final review before publication**
The Open Leadership Definition is [now available in GitHub for a final review](https://github.com/open-organization/open-leadership-definition/blob/main/open-leadership-definition.md). We are looking for feedback on grammar, typos, or other superficial bugs, because we plan to address any other significant feedback in the repository (using GitHub issues) after publication.

If you're interested in reviewing the planned release version, please do so by Thursday Aug 19 at 5pm EST. Our target publication date is Monday August 23, 2021.

Thank you again to all who participated in creating and reviewing this definition; it is truly a community-driven effort!

**Comment by semioticrobotic** [comment]
[It's released](https://theopenorganization.org/definition/open-leadership-definition/)!

Fantastic work on this, everyone. I sure do feel proud when I read it. Very excited to see how others will react to and use this fantastic tool.

Please see #121 if you're interested in helping promote the release.
